core/include/vecmem/utils/impl/copy.ipp

```diff
@@ -347,21 +401,32 @@ void copy::copy_views(std::size_t size,
 }
 
 template <typename TYPE>
-typename data::vector_view<TYPE>::size_type copy::get_size(
-    const data::vector_view<TYPE>& data) {
+std::vector<typename data::vector_view<TYPE>::size_type> copy::get_sizes(
+    const data::vector_view<TYPE>* data, std::size_t size) {
```

Comment:
Is `size` here the *outer size*?

Reply:
It is.

I introduced this **private** function mainly because in the `vecmem::copy::copy_views(...)` function we are receiving such `vecmem::data::vector_view` pointers already. And since I wanted to use `get_sizes(...)` there, it had to provide an interface that I could use.

The public functions just receive `vecmem::data::jagged_vector_view` and `vecmem::data::jagged_vector_buffer` references. So I thought that for the internal function I could go without writing too much documentation. :wink:

Reply:
Right, just trying to follow what's going on. I agree that private functionality doesn't have to be commented, although it would help me with reviewing of course 😅